_posts/2023-04-26-quarkus-3-0-final-released.adoc

```diff
@@ -0,0 +1,235 @@
+But Jakarta EE 10 also comes with new features for your favorite APIs.
+
+[NOTE]
+====
+Only the `javax.*` packages from Jakarta EE have changed.
+The ones from the JDK are still under `javax.*`.
+====
+
+=== Eclipse MicroProfile 6
+
+TODO
+
+=== Hibernate ORM 6
+
+This is one of the biggest changes in Quarkus 3: we upgraded to Hibernate ORM 6, more precisely 6.2.
+
+Hibernate ORM 6 is a major new version of Hibernate ORM and it comes with a lot of changes,
+with some being breaking changes.
+
+To learn more about the changes in Hibernate ORM 6, please have a look at the announcements for:
+
+- https://in.relation.to/2022/03/31/orm-60-final/[Hibernate ORM 6.0]
+- https://in.relation.to/2022/06/14/orm-61-final/[Hibernate ORM 6.1]
+- https://in.relation.to/2023/03/30/orm-62-final/[Hibernate ORM 6.2]
+
+Upgrading to Hibernate ORM 6 will require some efforts and testing, make sure you have a look at the <<upgrading>> section below.
+
+Some improvements have also been made to the extension itself such as:
+
+- The support of `StatementInspector` as `@PersistenceUnitExtension` managed bean
+- The ability to inject a https://docs.jboss.org/hibernate/orm/6.2/userguide/html_single/Hibernate_User_Guide.html#_statelesssession[`StatelessSession`] via CDI
+
+=== Hibernate Reactive 2
+
+Hibernate Reactive has been upgraded to version 2 to be compatible with Hibernate ORM 6.
+
+It is still in Beta for now but should be promoted to Final in the coming months.
+
+=== New Dev UI
+
+Quarkus 2 introduced the Dev UI which provides a web ui to use during development.
+Extensions could provide their own Dev UI pages to provide additional functionality.
+
+Quarkus 3 now has a new Dev UI which is more extensible and easier to use.
+It also has a new look and feel.
+
+The new Dev UI is the default but,
+given all the extensions have not yet migrated to the new one,
+you can still access the old one at `/q/dev-v1` but it will be dropped in a future version.
+
+We have a https://www.youtube.com/watch?v=sz5ihmA4gaE&list=PLsM3ZE5tGAVbyncLm7ue2V25cwFck7ew9[playlist] on Quarkus YouTube channel that shows off the new Dev UI, explains the new features and how to use and extend it.
+
+=== CLI plugins
+
+The Quarkus CLI was made extensible with plugins.
+Expect a lot of new features in the Quarkus CLI in the form of plugins in the near future.
+
+=== Upgrade tool
+
+Quarkus 3 comes with an upgrade tool that will radically simplify the upgrades from one version to another.
+
+You can learn more about it in the <<upgrading>> section.
+
+=== Support for Maven 3.9
+
+Maven 3.9 came with some breaking changes and the integration with Quarkus was not working properly.
+It is now fixed and you can use Maven 3.9 to build your Quarkus 3.0.0.CR1 applications.
+
+=== Gradle 8
+
+Our Gradle support has been upgraded to Gradle 8.
+
+=== Management network interface
+
+Until now, all the Quarkus endpoints were exposed on the same network interface.
```

Comment:
```suggestion
Hitherto, all the Quarkus endpoints were exposed on the same network interface.
```

Reply:
didnt apply this as hitherto isn't that commonly used I would say.

Reply:
We could even do `Heretofore` if you need more exocit 😆